src/actions/userActions.js

```diff
@@ -16,7 +16,7 @@ export const login = user =>
   async (dispatch) => {
     try {
       const response = await userApi.login({ user });
```

Comment:
```suggestion
const { data } = await userApi.login({ user });
```
You can destructure and then use the value on `await sessionService.saveUser(data);`